distributions/Dist_sj2879.py

```diff
@@ -1,21 +1,21 @@
+import math as math
 import numpy as np
 from .base_distribution import BaseDistribution
 
 class Dist_sj2879(BaseDistribution):
 	def __init__(self):
-		self.f_max = 1
-		self.x_min = -1
-		self.x_max = 1
-
+		self.f_max = 36
```

Comment:
This was the right value before you normalized. But now it is larger than it needs to be. Your normalized PDF is x**2/18, so f_max just needs to be 3**2/18 = 0.5

Reply:
Okay I just fixed this in my branch on the github desktop app, but for some reason, I can't send another pull request

Reply:
I am assuming I don't need to?